op-service/sources/types.go

```diff
@@ -61,6 +61,9 @@ type RPCHeader struct {
 	// ParentBeaconRoot was added by EIP-4788 and is ignored in legacy headers.
 	ParentBeaconRoot *common.Hash `json:"parentBeaconBlockRoot,omitempty"`
 
+	// RequestsHash was added by EIP-7685 and is ignored in legacy headers.
+	RequestsHash *common.Hash `json:"requestsHash" rlp:"optional"` // NOTE in geth the json tag is "requestsRoot"
+
```

Comment:
```suggestion
	RequestsHash *common.Hash `json:"requestsHash" rlp:"optional"`
```
This was fixed upstream https://github.com/ethereum/go-ethereum/pull/30926